src/modules/FSXARemoteApi.spec.ts

```diff
@@ -792,6 +792,20 @@ describe('FSXARemoteAPI', () => {
       const expectedURL = `${config.navigationServiceURL}/${config.contentMode}.${config.projectID}/by-seo-route/${encodedInitialPath}?depth=99&format=caas&all`
       expect(actualURL).toBe(expectedURL)
     })
+    it('should get available locales', async () => {
```

Comment:
This test seems to be implemented against an old implementation. Could you create a new test file for helper methods?